src/material/expansion/expansion-panel-header.scss

```diff
@@ -92,4 +92,9 @@
       content: '';
     }
   }
+  .mat-expansion-panel-header {
```

Comment:
What does this look like when it isn't expanded? I think that it might cause a double border at the bottom. Maybe the border should be on the content element instead?

Reply:
fixed